qiskit/result/utils.py

```diff
@@ -10,33 +10,35 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Utility functions for working with Results."""
+"""Utility functions for working with Result counts."""
```

Comment:
I guess this was referring to `Result` as the class. Given that `Counts` is a related-but-different class, I think this change is more confusing that helping.

Reply:
Makes sense. Done.